### DISCUSSION TITLE
Avoid edges and its labels overlapping in Gephi

### DISCUSSION THREAD

**Discussion by chee87** [gephi/gephi#2648] [Q&A]
Hi,

I have created a graph using Gephi. Two of the nodes in the graph are connected with two different edges, each represent different relationships. However, the edges and its labels are overlapped (see below image). How do I avoid the overlapping of the two edges?

Thanks.

<img width="157" alt="image" src="https://user-images.githubusercontent.com/90243545/192578183-1f6c93b2-be36-46c1-a2bb-c60d4ffd31c1.png">

**Comment by qkxie** [comment]
Did you find the solution? My workaround method is to decrease the font size of label and use the layout called "LabelAdjust"
